redis.go

```diff
@@ -1,4 +1,5 @@
 // Package redis implements a redis client for k6.
+// nolint: exhaustruct
```

Comment:
I am pretty sure we will not enable this linter after all

also (and this will be enforced in go 1.19) comment directives (comments not for humans) are supposed to be without spaces between `//` and the namespace (`nolint` in this case) and no space around the `:` you can see https://github.com/golangci/golangci-lint/issues/1658 and the linked issues.

Reply:
Thank you so much for the heads-up 🙇🏻 Will adjust accordingly, and remove the linter from the repo's config.

Reply:
Fixed 👍🏻 Thanks for the heads-up 🙇🏻

Reply:
Either github is showing some cached version or it isn't fixed ;)

Reply:
Damn, I missed that one, sorry for that 🙇🏻